src/main/java/nl/moreniekmeijer/backendspringboottechiteasycontroller/controllers/ExceptionController.java

```diff
@@ -1,6 +1,26 @@
 package nl.moreniekmeijer.backendspringboottechiteasycontroller.controllers;
 
-import org.springframework.web.bind.annotation.RestController;
+import nl.moreniekmeijer.backendspringboottechiteasycontroller.exceptions.NameTooLongException;
+import nl.moreniekmeijer.backendspringboottechiteasycontroller.exceptions.RecordNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
 
+@ControllerAdvice
 public class ExceptionController {
+    @ExceptionHandler(value = RecordNotFoundException.class)
+    public ResponseEntity<Object> handleRecordNotFoundException(RecordNotFoundException exception) {
+        return new ResponseEntity<>(exception.getMessage(), HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler(value = IndexOutOfBoundsException.class)
```

Comment:
Java heeft een eigen klasse die al IndexOutOfBoundsException heet. daardoor lijkt de code te werken Maar in je TelevisionController gooi je je eigen indexoutofboundsexception. maar hier wordt die van de standaard java gebruikt. 

Je zou dit " import nl.moreniekmeijer.backendspringboottechiteasycontroller.exceptions.IndexOutOfBoundsException; " bovenin kunnen toevoegen om te zeggen dat het om jouw eigen klasse gaat die moet worden afgevangen ipv die van java.